### PR TITLE
add sapporo docs

### PR DESCRIPTION
**Pull request by JeffreyThiessen**
## What does this PR do and why?
_Describe in detail what your merge request does and why._
Reliant on #507

Removes the documentation for ga4gh starter kit
Adds documentation for sapporo ga4gh wes implementation

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

Start the docs locally

```bash
cd docs-site
pnpm build
npm run serve
```

Check that the new docs page looks correct

Then follow the instructions as written and verify that everything works and runs correctly. If you need to deviate from the instructions please indicate in the PR review for corrected instructions.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: docs-site/docs/development/integration/ga4gh_wes_sapporo_setup.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ga4gh_wes:
+  server_url_endpoint: 'http://localhost:1122/'
+```
+
+### Setup Sapporo (WES implementation)
+
+Download and run the [PHAC-NML Sapporo](https://github.com/phac-nml/sapporo-service) fork in dev docker mode
+
+```bash
+# Clone and checkout the irida-next branch
```
Comment (joshsadam) [inline docs-site/docs/development/integration/ga4gh_wes_sapporo_setup.md:39]: **question:** Do I do this in my home directory?

File: docs-site/docs/development/integration/ga4gh_wes_sapporo_setup.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 1
+id: ga4gh_wes_sapporo_setup
+title: GA4GH WES Sapporo Setup
+---
+
+## Prerequisites
+
+You will need to have [Docker](https://docs.docker.com/get-docker/) installed for managing Sapporo
```
Comment (joshsadam) [inline docs-site/docs/development/integration/ga4gh_wes_sapporo_setup.md:9]: **suggestion:** Can you specify the minimum version.  FYI, it's not `Docker version 24.0.5, build 24.0.5-0ubuntu1`

File: docs-site/docs/development/integration/ga4gh_wes_sapporo_setup.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## How to set up a Sapporo GA4GH WES instance for development
+
+Note: This should only be used for development purposes, use a production WSGI server for production environments.
+
+### Configure IRIDA Next
+
+Set active storage service to `:local` in `config/environments/development.rb`
+
```
Comment (joshsadam) [inline docs-site/docs/development/integration/ga4gh_wes_sapporo_setup.md:18]: **question:** Should these be set in there by default?

**Review by ChrisHuynh333 — DISMISSED** [review]
Following the instructions worked for me. Only comment is that I did not have `docker compose` so I had to figure that part out in case you guys believe that needs to be explicitly added to these docs.

**Review by ksierks — DISMISSED** [review]
Worked for me after fixing my docker permissions to not run commands with sudo. Thanks!

File: docs-site/docs/development/integration/ga4gh_wes_sapporo_setup.md (hunk 1 above)
Comment (JeffreyThiessen) [reply docs-site/docs/development/integration/ga4gh_wes_sapporo_setup.md:66]: Clarification added in https://github.com/phac-nml/irida-next/pull/538/commits/71bfb20d95ffc5c382b1ffca5f2f217d745d2fb8

File: docs-site/docs/development/integration/ga4gh_wes_sapporo_setup.md (hunk 2 above)
Comment (JeffreyThiessen) [reply docs-site/docs/development/integration/ga4gh_wes_sapporo_setup.md]: Expected version number added in https://github.com/phac-nml/irida-next/pull/538/commits/3f2206b4ca63bcb6322baaf4c0d7725c73ed1fea

**Review by joshsadam — DISMISSED** [review]
Looks good to me now, thanks for the changes

**Comment by JeffreyThiessen** [comment]
> I did run into an issue getting to this page though, because when I clicked on Integrations in the dropdown menu, it would send me to the Front End Development page.
> 
> Eric thinks the slug in `docs-site/docs/development/integration/_catgegory.json` on line 6, needs to be changed to `/integration`

Fixed in https://github.com/phac-nml/irida-next/pull/538/commits/fc39107bf472f75d73608afa2d436318c845b6c4

**Pull request merged**